eclair-core/src/test/scala/fr/acinq/eclair/payment/PaymentInitiatorSpec.scala

```diff
@@ -128,7 +128,7 @@ class PaymentInitiatorSpec extends TestKitBaseClass with FixtureAnyFunSuiteLike
       Bolt11Invoice.Description("Some invoice"),
       Bolt11Invoice.PaymentSecret(randomBytes32()),
       Bolt11Invoice.Expiry(3600),
-      Bolt11Invoice.InvoiceFeatures(Features(bin"000001000000000000000000000000000100000100000000")) // feature 42
+      Bolt11Invoice.InvoiceFeatures(Features(bin"000001000000000000000000000000000100000100000000").invoiceFeatures()) // feature 42
```

Comment:
Now that we don't really control what happens in `.invoiceFeatures`, we should make this more explicit and use:
```suggestion
      Bolt11Invoice.InvoiceFeatures(Features[InvoiceFeature](Map(VariableLengthOnion -> Mandatory, PaymentSecret -> Mandatory), Set(UnknownFeature(42))))
```

Reply:
Done